modules/php/Game/Utils.php

```diff
@@ -57,7 +59,9 @@ public static function getActiveFirstPlayRandom()
   public static function isPartyInPlay()
   {
     $party_keeper_card_id = 16;
-    $party_keeper_card = self::getGame()->cards->getCard($party_keeper_card_id);
+    $party_keeper_card = array_values(
```

Comment:
Is this really needed? I feel the way we had was clearer.

Reply:
Yes, absolutely necessary bug fix: the getCard() function needs the internal database id of the card - which we don't know here (only when card is played from hand or so).
Here the "16" is our unique logical id of the Party keeper, so we need to do lookup via type / type_arg exactly like we do in the TwoKeeper goals.

Reply:
OK, understood. Thanks for the clarification.